test/extended/image_ecosystem/plugin.go

```diff
@@ -414,18 +417,46 @@ var _ = g.Describe("[image_ecosystem][jenkins][Slow] openshift pipeline plugin",
 		g.By("kick off the build for the jenkins ephermeral and application templates")
 		tag := []string{"openshift/jenkins-plugin-snapshot-test:latest"}
 		hexIDs, err := exutil.DumpAndReturnTagging(tag)
-		var jenkinsEphemeralPath string
-		var testingSnapshot bool
-		if len(hexIDs) > 0 && err == nil {
-			// found an openshift pipeline plugin test image, must be testing a proposed change to the plugin
-			jenkinsEphemeralPath = exutil.FixturePath("testdata", "jenkins-ephemeral-template-test-new-plugin.json")
-			testingSnapshot = true
+
+		// If the user has expressed an interest in local plugin testing by setting the
+		// SNAPSHOT_JENKINS_IMAGE environment variable, try to use the local image. Inform them
+		// either about which image is being used in case their test fails.
+		snapshotImagePresent := len(hexIDs) > 0 && err == nil
+		useSnapshotImage := os.Getenv(useLocalPluginSnapshotEnvVarName) != "" && snapshotImagePresent
+
+		//TODO disabling oauth until we can update getAdminPassword path to handle oauth (perhaps borrow from oauth integration tests)
+		newAppArgs := []string{exutil.FixturePath("testdata", "jenkins-ephemeral-template-no-oauth.json")}
```

Comment:
Will do. Definitely the intent of this PR to reduce the templates we need to maintain.